README.md

```diff
@@ -2,7 +2,7 @@
 
 <div align="center">
 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="./pictures/murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
- [OPTIONAL] Kindly remove the logo from the README.md file. If you have your personalized logo, you can use it, otherwise, I would advise you to remove it 👍 💻